DB/projects.json

```diff
@@ -3499,5 +3499,15 @@
         "tech": ["flutter", "hive", "dart"]
     }
   ]
-  }
+  },
+{
+	"github_url": "https://github.com/blindaks/Quiz-App",
+	"title": "Quiz App",
+	"description": "A quiz app based on front wnd development questions.",
+	"tech": [
+	        "html",
+	        "css",
+	        "javascript"
+	]
+}
```

Comment:
not here. check the docs https://projectshut.vercel.app/docs/ & sample code below

```
{
  "github_username": "YOUR_GITHUB_USERNAME",  
  "Projects": [
    {
      "github_url": "1st_PROJECT_LINK",
      "title": "PROJECT_NAME",
      "description": "PROJECT_DESCRIPTION",
      "tech": ["tech1", "tech2"]
    },
    {
      "github_url": "2nd_PROJECT_LINK",
      "title": "PROJECT_NAME",
      "description": "PROJECT_DESCRIPTION",
      "tech": ["tech1", "tech2"]
    }
  ]
}
```

Reply:
Do you want me to add "Projects:" line? So that I can put any future projects there?

Reply:
yes, in future you need to add ``` {
      "github_url": "2nd_PROJECT_LINK",
      "title": "PROJECT_NAME",
      "description": "PROJECT_DESCRIPTION",
      "tech": ["tech1", "tech2"]
    }```
inside the projects array

Reply:
Okay